misc/Dockerfile.nginx

```diff
@@ -1,7 +1,7 @@
 # NOTE: The builder part is nessessary to build headers more module
-FROM nginx:1.20.0 as builder
+FROM nginx:1.22.1 as builder
 
-ENV NGINX_VERSION 1.20.0
+ENV NGINX_VERSION 1.22.1
 ENV HEADERS_MORE_VERSION 0.33
```

Comment:
A new version, 0.34, is available.
https://github.com/openresty/headers-more-nginx-module/compare/v0.33...v0.34